frontend/src/components/MeetingRoom/UnreadMessagesBadge/UnreadMessagesBadge.tsx

```diff
@@ -12,21 +13,22 @@ export type UnreadMessagesBadgeProps = {
  * Displays a badge indicating the number of unread chat messages.
  * @param {UnreadMessagesBadgeProps} props - the props for the component
  *  @property {ReactElement} children - the ToolbarButton to be rendered
+ *  @property {boolean} isToolbarOverflowMenuOpen - (optional) indicates whether the overflow menu was opened
  * @returns {ReactElement} - The UnreadMessagesBadge component
  */
 const UnreadMessagesBadge = forwardRef(function UnreadMessagesBadge(
   props: UnreadMessagesBadgeProps,
   ref: ForwardedRef<HTMLSpanElement>
 ) {
-  const { children, ...rest } = props;
+  const { children, isToolbarOverflowMenuOpen, ...rest } = props;
   const { unreadCount } = useSessionContext();
-
+  const isMessageCountBadgeInvisible = unreadCount === 0 || isToolbarOverflowMenuOpen;
```

Comment:
Maybe `isInvisible` or `isUnreadMessagesBadgeInvisible` instead? I don't think we reference `MessageCountBadge` anywhere.